Makefile

```diff
@@ -30,20 +30,29 @@ export CGO_ENABLED:=0
 export GO111MODULE:=on
 export GOPROXY?=https://proxy.golang.org/
 
```

Comment:
WDYT about we add `.DEFAULT_GOAL:=help`? Then, if we run `make` it will how the help as well,